src/sentry/monitors/endpoints/base.py

```diff
@@ -4,10 +4,10 @@
 
 from rest_framework.request import Request
 
+from sentry.api.authentication import ApiKeyAuthentication, DSNAuthentication, TokenAuthentication
```

Comment:
Do we really need to support `ApiKeyAuthentication`? [Our documentation states that these are a legacy means of authentication.](https://docs.sentry.io/api/auth/#api-keys)

Reply:
I need to understand if anyone was using the APIKeyAuth for this or not :(